index.js

```diff
@@ -4,18 +4,22 @@ var redis = Redis.createClient({host: process.env.REDIS_URL || 'redis'});
 var express = require('express');
 var cors = require('cors');
 var app = express();
+var since = redis.get('npm:latest_seq') || 'now'
 
 var changes = new ChangesStream({
   db: 'https://replicate.npmjs.com/registry',
   include_docs: true,
-  since: 'now'
+  since: since
 });
 
 changes.on('data', function (change) {
   var name = change.doc.name
+  var version = change.doc['dist-tags'].latest
   if(name){
-    console.log(name)
+    console.log(change.seq, name, version)
```

Comment:
Want to 2x check with @katzj first that we don't rely on the logging format for anything -- this new format will look like:

```
11074016 'simple-element' '0.4.8'
11074017 'vue-draggable-resizable-cbim' '0.0.8'
11074018 'uikit' '3.6.8'
...
```

Reply:
We don't look at these logs at all right now... will add my usual that it's way better to always do log messages with key=value rather than just keys in some order. makes parsing or querying way easier if we ever need or want to